resources/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "BitGoWalletRecoveryWizard",
   "author": "BitGo, Inc.",
   "description": "A UI-based desktop app for BitGo Recoveries",
-  "version": "2.8.8",
+  "version": "2.2.3",
```

Comment:
we decreased the verison here?

Reply:
I realized when you create a build/executable, the version you specify gets updated in the package.json. The 2.2.3 was just an example version in the readme. I changed it back